mmdet3d/ops/furthest_point_sample/__init__.py

```diff
@@ -1,3 +1,4 @@
-from .furthest_point_sample import furthest_point_sample
+from .furthest_point_sample import (furthest_point_sample,
+                                    furthest_point_sample_with_dist)
 
-__all__ = ['furthest_point_sample']
+__all__ = ['furthest_point_sample', 'furthest_point_sample_with_dist']
```

Comment:
`furthest_point_sample_with_dist` -> `furthest_point_sample_by_dist`

Reply:
The name of the function means computing furthest points sampling with distance inputs.